lib/modules/survey_detail/components/content.dart

```diff
@@ -0,0 +1,67 @@
+part of '../survey_detail_module.dart';
+
+class Content extends StatelessWidget {
+  const Content({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Screen(
+      body: Stack(
+        fit: StackFit.expand,
+        children: [
+          Image(
+            image: Assets.images.mainBackgroundDimmed,
+            fit: BoxFit.fill,
+          ),
+          SafeArea(
+            child: Column(
+              children: [
+                NavigationBar(),
+                Expanded(
+                  child: Container(
+                    margin: const EdgeInsets.fromLTRB(20, 0, 20, 0),
+                    child: Column(
+                      children: [
+                        const Text(
+                          "Working from home Check-In",
+                          style: TextStyle(
+                            color: Colors.white,
+                            fontSize: 34,
+                          ),
```

Comment:
Just a suggestion, instead of creating `TextStyle` manually many times, we can define each `TextStyle` type in `Theme` and use it. 🙏

Reply:
@manh-t noted it. Let's me update it in refactor progress 🙏